src/case_gather/services.py

```diff
@@ -17,12 +17,21 @@ def update_cases_by_session_number(parliament_session):
         #  'number', 'name', 'case_type', 'althingi_status'
         #  'rel_cases', 'subjects', 'session'
         case_number_int = int(case['number'])
-        if not Case.objects.filter(number=case_number_int).exists():
+        print(case_number_int)
+        print(case['name'])
+        case_exists = Case.objects.filter(number=case_number_int, parliament_session=parliament_session)
+        # We find the status from the althingi status
+        althingi_status = case['althingi_status']
+        print(case_exists.exists())
```

Comment:
probably worth it to give some context with the print statements.

`print(Althingi case number > + case_number_int)`
`print(Althingi case name > + case['name'])`
`print(Did we already have this case? > case_exists.exists())`